fluss-common/src/main/java/com/alibaba/fluss/config/ConfigOptions.java

```diff
@@ -1073,7 +1082,9 @@ public class ConfigOptions {
                                     + "configured as 3, then partitions 20241111, 20241112, 20241113 will be pre-created. "
                                     + "If any one partition exists, it'll skip creating the partition. "
                                     + "The default value is 2, which means 2 partitions will be pre-created. "
-                                    + "If the 'table.auto-partition.time-unit' is 'DAY'(default), one precreated partition is for today and another one is for tomorrow.");
+                                    + "If the 'table.auto-partition.time-unit' is 'DAY'(default), one precreated partition is for today and another one is for tomorrow."
+                                    + "For a partition table with multiple partition keys, pre-create is unsupported and this value must be strictly less than or equal to 0."
```

Comment:
It is not friendly to force users to set another config. I think we can set it to 0 automatically by server. And add a description here, "For a partition table with multiple partition keys, pre-create is unsupported and will be set to 0 automatically when creating table if it is not explicitly specified."

Regarding implementation, I think we can add the logic in `CoordinatorService#applySystemDefaults` that set the config to 0 if it's multi-level partition table and this config is not explicitly specified.